examples/misc/lib/effective_dart/design_good.dart

```diff
@@ -0,0 +1,281 @@
+// ignore_for_file: type_annotate_public_apis, unused_element, unused_local_variable
+// #docplaster
+
+import 'dart:async';
+import 'dart:collection';
+import 'dart:isolate';
+import 'dart:math';
+import 'dart:typed_data';
+
+import 'package:dartlang_examples_util/ellipsis.dart';
+
+dynamic element, key, value;
+ByteBuffer bytes;
+DateTime dateTime;
+List list;
+Map map;
+String string;
+StreamSubscription subscription;
+
+void miscDeclAnalyzedButNotTested() {
+  (Iterable errors, Iterable monsters) {
+    // #docregion code-like-prose
+    // "If errors is empty..."
+    if (errors.isEmpty) {/*-...-*/}
+
+    // "Hey, subscription, cancel!"
+    subscription.cancel();
+
+    // "Get the monsters where the monster has claws."
+    monsters.where((monster) => monster.hasClaws);
+    // #enddocregion code-like-prose
+  };
+
+  (entryPoint, message, elements, pattern) {
+    // #docregion omit-verb-for-bool-param
+    Isolate.spawn(entryPoint, message, paused: false);
+    new List.from(elements, growable: true);
+    new RegExp(pattern, caseSensitive: false);
+    // #enddocregion omit-verb-for-bool-param
+  };
+
+  (DoubleLinkedQueue queue, window, connection) {
+    // #docregion verb-for-func-with-side-effect
+    list.add(ellipsis);
+    queue.removeFirst();
+    window.refresh();
+    connection.downloadData();
+    // #enddocregion verb-for-func-with-side-effect
+  };
+
+  () {
+    // #docregion noun-for-func-returning-value
+    list.elementAt(3);
+    string.codeUnitAt(4);
```

Comment:
Consider adding `list.firstWhere(test)` as an example that doesn't end in `At`.

Reply:
Good suggestion. Done.